polclassifier/interface/main.py

```diff
@@ -204,6 +250,37 @@ def train_evaluate_model_knn(split_ratio: float = 0.2, perform_search: bool = Fa
     return model, accuracy
 
 
+def load_speeches(min_word_count=400, sample_size=1000, parties_to_exclude=[], speeches_per_party = 20):
```

Comment:
Do we need this function? Shouldn't we be saving the speeches into `smaller_data_sample_text.csv` (and only the middle 200 words)?